### DISCUSSION TITLE
Support multi send to

### DISCUSSION THREAD

**Discussion by vevedh** [Budibase/budibase#7576] [Contributing]
Replace :
email: Joi.string().email()

by 

email: Joi.string().email({
      multiple: true,
  }), 

in packages\worker\src\api\routes\global\email.js

Result :  

<img width="294" alt="image" src="https://user-images.githubusercontent.com/1430389/187998955-36fe087f-48ff-4bb6-9d02-98b154a831b4.png">

**Comment by melohagan** [comment]
Nice one! Does the email actually get sent to both users?

**Reply by vevedh** [reply]
Yes, sent to both users
FR:Oui il est bien envoyé, merci

**Reply by melohagan** [reply]
Brilliant! I'm sure a PR would be appreciated if you want to link it to this issue: https://github.com/Budibase/budibase/issues/4385
